nilearn/decoding/space_net.py

```diff
@@ -0,0 +1,1242 @@
+
+    l1_ratios : float or list of floats in the interval [0, 1];
+    optinal (default .5)
+        Constant that mixes L1 and spatial prior terms in penalization.
+        l1_ratio == 1 corresponds to pure LASSO. The larger the value of this
+        parameter, the sparser the estimated weights map. If list is provided,
+        then the best value will be selected by cross-validation.
+
+    alphas : float or list of floats, optional (default None)
+        Choices for the constant that scales the overall regularization term.
+        This parameter is mutually exclusive with the `n_alphas` parameter.
+        If None or list of floats is provided, then the best value will be
+        selected by cross-validation.
+
+    n_alphas : int, optional (default 10).
+        Generate this number of alphas per regularization path.
+        This parameter is mutually exclusive with the `alphas` parameter.
+
+    eps : float, optional (default 1e-3)
+        Length of the path. For example, ``eps=1e-3`` means that
+        ``alpha_min / alpha_max = 1e-3``
+
+    alpha_min : float, optional (default None)
+        Minimum value of alpha to consider. This is mutually exclusive with the
+        `eps` parameter.
+
+    mask : filename, niimg, NiftiMasker instance, optional default None)
+        Mask to be used on data. If an instance of masker is passed,
+        then its mask will be used. If no mask is it will be computed
+        automatically by a MultiNiftiMasker with default parameters.
+
+    target_affine : 3x3 or 4x4 matrix, optional (default None)
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    target_shape : 3-tuple of integers, optional (default None)
+        This parameter is passed to image.resample_img. Please see the
+        related documentation for details.
+
+    low_pass : False or float, optional, (default None)
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    high_pass : False or float, optional (default None)
+        This parameter is passed to signal. Clean. Please see the related
+        documentation for details
+
+    t_r : float, optional (default None)
+        This parameter is passed to signal.clean. Please see the related
+        documentation for details
+
+    screening_percentile : float in the interval [0, 100]; Optional (
+    default 20)
+        Percentile value for ANOVA univariate feature selection. A value of
+        100 means 'keep all features'. This percentile is is expressed
+        w.r.t the volume of a standard (MNI152) brain, and so is corrected
+        at runtime to correspond to the volume of the user-supplied mask
+        (which is typically smaller).
+
+    standardize : bool, optional (default True):
+        If set, then the data (X, y) are centered to have mean zero along
+        axis 0. This is here because nearly all linear models will want
```

Comment:
The doc for `normalize` is zombie code. This parameter is not used anymore. Removed.